src/test/java/com/google/gcp_variant_transforms/data/SampleHeaderv4.2.vcf

```diff
@@ -0,0 +1,29 @@
+##fileformat=VCFv4.2
```

Comment:
How about creating a `resources` folder in the test module and it contains all test vcf files?

Reply:
Currently, SampleHeaderv4.2.vcf is inside a folder called data in the test module, do you mean renaming this folder to resources?

Reply:
https://github.com/BoscoSuen/gcp-variant-transforms-java/tree/vc2BQpart2/src/test I added all test files in the resources folder. But it's up to you, not a big problem.

Reply:
Is there a benefit or reason for having the resources folder immediately after src/test versus src/test/java/com.google.gcp_variant_transforms/data? We can certainly move this